CHANGELOG.md

```diff
@@ -1,12 +1,34 @@
 # Changelog
 
+## 5.2.0
+
+### Features
+
+- [#832](https://github.com/okta/okta-auth-js/pull/832) Supports rotating refresh tokens
+
+### Bug Fixes
+
+- [#832](https://github.com/okta/okta-auth-js/pull/832) Fixes issues with refresh tokens
+
+## 5.1.1
```

Comment:
Thanks for adding all those changelogs, I'll just close my PR https://github.com/okta/okta-auth-js/pull/835